docs/workflow/snippets/prereqs-desktop.mdx

```diff
@@ -0,0 +1,9 @@
+:::info Important
+While extending VertiGIS Studio Workflow for Desktop can easily be achieved, you require an installed Workflow runtime module to make your custom activities work with VertiGIS Studio Desktop.
+:::
+
+To ease using your custom activities when authoring of Workflows with Workflow Designer, you may create method stubs for your activities and publish them to make them, as described in [the corresponding chapter](sdk-net-register-activities).
```

Comment:
This link is breaking this build. Looks like it needs a `../`

Reply:
The problem is, that this is an included file (referencing a file in its parent directory). Adding a "../" breaks the link and references a non-existing file, starting to look for the file in the including file's parent directory...

Reply:
I got it to work by making it `../sdk-net-register-activities.mdx`. The `.mdx` extension is required.

I think we only use the `/docs/` rooted paths for React components where we have code that looks like this
`link={useBaseUrl("docs/mobile/sdk-overview")}`

Reply:
Got it working, using the full path like `/docs/workfllow/sdk-net-register-activities`. Will be going the `.mdx` suffix though.